lib/helper/QubitHelper.php

```diff
@@ -174,6 +178,101 @@ function render_show($label, $value, $options = [])
 contents;
 }
 
+function render_b5_show_field_css_classes($options = [])
+{
+    return 'row g-0';
+}
+
+function render_b5_show_subfield_css_classes($options = [])
+{
+    return 'd-flex flex-wrap';
+}
+
+function render_b5_show($label, $value, $options = [])
+{
+    $tag = 'div';
+    $cssClasses = 'field';
+    if (!isset($options['isSubField'])) {
+        $cssClasses .= ' '.render_b5_show_field_css_classes($options);
+    } else {
+        $cssClasses .= ' '.render_b5_show_subfield_css_classes($options);
+    }
+
+    $labelContainer = render_b5_show_label($label, $options);
+    $valuecontainer = render_b5_show_value($value, $options);
+
+    return render_b5_show_container(
+        $tag, $labelContainer.$valuecontainer, $cssClasses, $options
+    );
+}
+
+function render_b5_show_container($tag, $content, $cssClasses = '', $options = [])
+{
+    $cssClass = $cssClasses ? ' class="'.$cssClasses.'"' : '';
+
+    return <<<contents
+<{$tag}{$cssClass}>{$content}</{$tag}>
+contents;
```

Comment:
Could this be `return "<{$tag}{$cssClass}>{$content}</{$tag}>";` directly? I hate heredocs.